lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -124,25 +124,24 @@
 
 import re
 from time import sleep
+from ansible import constants as C
 from ansible.module_utils.network.nxos.nxos import load_config, run_commands
 from ansible.module_utils.network.nxos.nxos import nxos_argument_spec, check_args
 from ansible.module_utils.basic import AnsibleModule
 
 
 def check_ansible_timer(module):
     '''Check Ansible Timer Values'''
-    msg = "The 'timeout' provider param value for this module to execute\n"
-    msg = msg + 'properly is too low.\n'
-    msg = msg + 'Upgrades can take a long time so the value needs to be set\n'
-    msg = msg + 'to the recommended value of 500 seconds or higher in the\n'
-    msg = msg + 'ansible playbook for the nxos_install_os module.\n'
-    msg = msg + '\n'
-    msg = msg + 'provider: "{{ connection | combine({\'timeout\': 500}) }}"'
-    data = module.params.get('provider')
+    command_timer = C.PERSISTENT_COMMAND_TIMEOUT
+    connect_timer = C.PERSISTENT_CONNECT_TIMEOUT
```

Comment:
@mattclay I am simply trying to make sure that users are setting the ANSIBLE_ PERSISTENT_COMMAND_TIMEOUT and ANSIBLE_ PERSISTENT_CONNECT_TIMEOUT high enough to be used with this module.  This module controls Nexus upgrades so these need to be set in a way to avoid timeout issues.  Can I change these timers inside an Ansible module directly?  I thought that they were checked/used before we get to the module code.